samples/sample-infra/azure-services.bicep

```diff
@@ -0,0 +1,141 @@
+param location string
+param redisCacheName string
+param eventHubNamespace string
+param eventHubName string
+param eventHubSku string
+param containerName string
+param envResourceNamePrefix string
+
+/* ###################################################################### */
+// Create storage account for function app prereq
+/* ###################################################################### */
+resource azStorageAccount 'Microsoft.Storage/storageAccounts@2021-08-01' = {
```

Comment:
Should we move storage account and managedEnvironment to azure-function.bicep? Those three resources together complete an azure-function deployment. https://github.com/Azure/azure-functions-dapr-extension/blob/dev/e2e-test-support/test/EndToEnd/Tests/Framework/Scripts/DeployFunctionsOnAca.bicep

Reply:
azStorageConnectionString is being used in functions as well as it is required for Azure EventHub, and resource creation is as follows

1. storage account
2. dapr component setup
3. function creation

If I merge, storage account in function creation bicep, dapr component will depend on function creation, which is not correct I believe. So, having it in service creation bicep is better.

Reply:
Got it. Thanks for clarifying the thought process. LGTM.